### PR TITLE
PoC: use Google API linter on GH

### PR DESCRIPTION
**Pull request by moritzzimmer**
Added https://linter.aip.dev/ to makefile and PR workflow using default rules.

This results in something like this

```
2021/02/24 15:29:44 found problems during linting
+------------------------------------+------------------+----------------+
|                RULE                | TOTAL VIOLATIONS | VIOLATED FILES |
+------------------------------------+------------------+----------------+
| core::0140::abbreviations          |                2 |              1 |
| core::0131::response-message-name  |                3 |              3 |
| core::0131::request-name-required  |                4 |              4 |
| core::0127::http-annotation        |                4 |              4 |
| core::0131::request-unknown-fields |                4 |              4 |
| core::0131::method-signature       |                4 |              4 |
| core::0203::optional               |                4 |              3 |
| core::0203::required               |                9 |              4 |
| core::0191::java-outer-classname   |                9 |              9 |
| core::0192::has-comments           |               15 |              4 |
+------------------------------------+------------------+----------------+
Linted 9 proto files
make: *** [lint] Error 1
Makefile:65: recipe for target 'lint' failed
```

No updating of PR comments with step results so far.

### CODE REVIEW

File: stroeer/core/v1/article.proto [hunk 1]
```diff
@@ -87,11 +87,11 @@ message Metadata {
 
   // Manually set editorial date ('Gültig von') at which the article is valid to deliver on
   // digital platforms in seconds of UTC time since Unix epoch.
-  google.protobuf.Timestamp valid_from = 2;
+  google.protobuf.Timestamp valid_time = 2;
```
Comment (thisismana) [inline stroeer/core/v1/article.proto:90]: this will be fixed in #160

File: .github/workflows/pull-request.yml [hunk 2]
```diff
@@ -6,6 +6,18 @@ on:
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-go@v2
+
+      - run: go get -u github.com/googleapis/api-linter/cmd/api-linter
+
+      - name: Google API linter
+        run: api-linter --output-format stroeer/core/v1/article.proto
```
Comment (thisismana) [inline .github/workflows/pull-request.yml:19]: section no?

**Review by thisismana — CHANGES_REQUESTED** [review]

**Comment by bobaaaaa** [comment]
cool tool but the DX is bad :/

**Comment by moritzzimmer** [comment]
@bobaaaaa @thisismana I think we can keep this tool in our toolchain (makefile, gh workflows) for (optional) linting

**Pull request merged**